templates/views/contact.pug

```diff
@@ -0,0 +1,32 @@
+extends ../layouts/base
+
+block content
+  #contact.w-100.absolute.tc.bg-gray.z-1
+    .mxwi6.dib.tc.w-100.pb4
+      h2.tc.light-gray Contact us
+      
+      #address 
+      h3.red Address
+      .silver Gower Str, Bloomsbury, London, WC1E 6BT
+      
+      #phone 
+      h3.red Phone
+      .silver 000 0000 0000
```

Comment:
since this is wrong and we don't know if the client even wants to have a phone number displayed, I would comment this out for now